pkg/slack/events/helpdesk/helpdesk-message.go

```diff
@@ -0,0 +1,45 @@
+package helpdesk
+
+import (
+	"strings"
+
+	"github.com/openshift/ci-tools/pkg/slack/events"
+	"github.com/sirupsen/logrus"
+	"github.com/slack-go/slack/slackevents"
+)
+
+const (
+	channelId      = "CBN38N3MW"
+	dptpHelpdeskId = "<@STSR51Q76>"
+)
+
+// Handler returns a handler that knows how to respond to new messages
+// in forum-testplatform channel that mention @dptp-helpdesk.
+func Handler() events.PartialHandler {
+	return events.PartialHandlerFunc("helpdesk",
+		func(
+			callback *slackevents.EventsAPIEvent,
+			logger *logrus.Entry) (handled bool, err error) {
```

Comment:
this spacing looks a little off. try:
```suggestion
		func(callback *slackevents.EventsAPIEvent, logger *logrus.Entry) (handled bool, err error) {
```